daprdocs/content/en/reference/components-reference/supported-name-resolution/setup-nr-consul.md

```diff
@@ -3,7 +3,7 @@ type: docs
 title: "HashiCorp Consul"
```

Comment:
"HashiCorp Consul Resolution Component" to be consistent with the other titles, or remove "  Resolution Component" on all of them.